I_dont_have_vinaigrette.py

```diff
@@ -0,0 +1,43 @@
+import random
+import calendar
+import datetime
+
+
+
+def  print_random_date(s_date, e_date):
```

Comment:
Add type hints and return types - read here about how its done - https://docs.python.org/3/library/typing.html

Reply:
datetime.date can be written as a variable type and return type?